<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsDialog</class>
 <widget class="QDialog" name="StatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 40, 59);</string>
  </property>
  <widget class="QLineEdit" name="filenameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cyclesText">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="arquitecturaText">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipoRText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipoIText">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipoBText">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipoSText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipoJText">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipoUText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cantidad de instrucciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>384</y>
     <width>427</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>mononoki Nerd Font</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MÃ¡s detalle sobre las instrucciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>459</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>StatsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
